velox/type/Conversions.h

```diff
@@ -59,11 +59,63 @@ struct Converter {
   }
 };
 
-/// Presto compatible cast of strings to boolean. There is a set of strings
-/// allowed to be casted to boolean. These strings are `t, f, 1, 0, true, false`
-/// and their upper case equivalents. Casting from other strings to boolean
+/// This function casts a string to a boolean value. There is a set of strings
+/// allowed to be casted to boolean. Casting from other strings to boolean
 /// throws.
-Expected<bool> castToBoolean(const char* data, size_t len);
+/// @tparam TPolicy The specified cast policy. PrestoCastPolicy and
+/// LegacyCastPolicy allow `t, f, 1, 0, true, false` and their upper case
+/// equivalents. SparkCastPolicy allows all strings that they allow plus `y, n,
```

Comment:
nit: may drop 'The specified cast policy.'
nit: SparkCastPolicy additionally allows 'y, n, yes, no' and their upper case equivalents.

Reply:
Thanks for the suggestion. Updated.